feature/bgp/policybase/otg_tests/comm_ext_match_action_test/README.md

```diff
@@ -0,0 +1,181 @@
+      * actions/bgp-actions/set-community/config/method = REFERENCE
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * For each policy-definition created, run a subtest (RT-7.8.2.x-neighbor-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify routes are received on ATE port 1 for all prefixes (since all routes are accepted by policies).
+    * Verify expected communities are present in ATE.
+    * Verify expected communities are present in DUT state.
+      * Do not fail test if this path is not supported, only log results
+      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+
+  * For each policy-definition created, run a sub-test (RT-7.8.2.x-peer-group-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify expected communities are present in ATE.
+    * Verify expected communities are present in DUT state.
+      * Do not fail test if this path is not supported, only log results
+      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+
+    * Expected communities
+      |              | add_std_comms                                 | match_and_add_std_comms           |
+      | ------------ | --------------------------------------------- | --------------------------------- |
+      | prefix-set-1 | [ 10:10, 20:20, 30:30 ]                       | none                              |
+      | prefix-set-2 | [ 10:10, 20:20, 30:30, 5:5, 6:6 ]             | [ 10:10, 20:20, 30:30, 5:5, 6:6 ] |
+      | prefix-set-3 | [ 10:10, 20:20, 30:30, 50:500000, 60:600000 ] | [ 50:500000, 60:600000 ]          |
+
+* RT-7.8.3 - Create policy to set ext-community for various match criteria using OC release 3.x
```

Comment:
This are not extended communities.
The only extended community type we care about is link-bandwidth. so perhaps this test is not needed.

Reply:
Happy to remove, please see [this comment](https://github.com/openconfig/featureprofiles/pull/2654#discussion_r1501209150) and we can decide to keep or remove.

Reply:
removed